geonode/templates/base.html

```diff
@@ -151,6 +144,28 @@
       <p>Please use a <strong>modern browser</strong> like Mozilla Firefox, Google Chrome or Safari.</p>
     </div>
```

Comment:
What I meant is, you should probably ask Geonode developers for review wether to improve existing codes or change it. Because I don't know what's best for these and perhaps they have their own reasons/standard to implement this.